### PR TITLE
[Enhancement](streamload) print profile for streamload

### LINKED ISSUE #17169: [Enhancement] print profile for streamload
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Currently only BrokerLoad and InsertInto support load profile.
We need print profile info after StreamLoad, too.

What is profile? Why we need that? see [link](https://doris.apache.org/zh-CN/docs/dev/admin-manual/query-profile?_highlight=profile#profile%E5%8F%82%E6%95%B0%E8%A7%A3%E6%9E%90)

### Solution

When `set global enable_profile=true`, Streamload is actually profiled. However, this profile result is shown nowhere. We should print it out in the following function if is enabled.

```
be/src/runtime/plan_fragment_executor.cpp
void PlanFragmentExecutor::close()
```

Note that StreamLoad could be more frequent than other load jobs. Hence, to avoid too many profiles print in log, we need a control flag in be/src/common/config.h to allow the user turn on/off the print.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gitccl**
# Proposed changes

Issue Number: close #17169

## Problem summary

When both `enable_profile` and `enable_stream_load_profile_log` is true, stream load profile is printed to the log:
```
I0322 19:49:53.000802 2990646 plan_fragment_executor.cpp:534] Fragment bc419b20fd50e30a-5860a2ee0c8c9998:(Active: 3.724ms, non-child: 0.00%)
   - FragmentCpuTime: 732.917us
   - RowsProduced: 2
  OlapTableSink:(Active: 3.899ms, non-child: 100.00%)
     - CloseWaitTime: 3.206ms
     - MaxAddBatchExecTime: 2.719ms
     - NonBlockingSendTime: 1.106ms
       - NonBlockingSendWorkTime: 68.512us
         - SerializeBatchTime: 13.590us
     - NumberBatchAdded: 1
     - NumberNodeChannels: 1
     - OpenTime: 581.217us
     - RowsFiltered: 0
     - RowsRead: 2
     - RowsReturned: 2
     - SendDataTime: 33.451us
       - WaitMemLimitTime: 0.000ns
     - TotalAddBatchExecTime: 2.719ms
     - ValidateDataTime: 5.081us
  VFILE_SCAN_NODE (id=0):(Active: 832.459us, non-child: 22.35%)
     - UseSpecificThreadToken: False
     - AcuireRuntimeFilterTime: 5.587us
     - BytesDecompressed: 0
     - BytesRead: 21.00 B
     - DecompressTime: 0.000ns
     - FileReadTime: 13.283us
     - GetNextTime: 11.849us
     - MaxScannerThreadNum: 1
     - NumScanners: 1
     - PeakMemoryUsage: 0
     - PreAllocFreeBlocksNum: 8
     - ProjectionTime: 0.000ns
     - RowsRead: 2
     - RowsReturned: 2
     - RowsReturnedRate: 2.40 K/sec
     - ScannerWorkerWaitTime: 38.946us
     - TotalReadThroughput: 0
    VScanner:
       - PerScannerRunningTime: [205.858us, ]
       - PerScannerRowsRead: [2, ]
       - FileScannerCastInputBlockTime: 30.539us
       - FileScannerConvertOuputBlockTime: 51.586us
       - FileScannerFillMissingColumnTime: 0.000ns
       - FileScannerFillPathColumnTime: 0.000ns
       - FileScannerGetBlockTime: 48.873us
       - FileScannerPreFilterTimer: 0.000ns
       - NewlyCreateFreeBlocksNum: 0
       - ScannerBatchWaitTime: 91.000ns
       - ScannerConvertBlockTime: 0.000ns
       - ScannerCpuTime: 362.469us
       - ScannerCtxSchedCount: 4
       - ScannerFilterTime: 87.000ns
       - ScannerGetBlockTime: 205.329us
       - ScannerPrefilterTime: 0.000ns
       - ScannerSchedCount: 1
      MemoryUsage:
         - FreeBlocks: 704.00 KB
         - QueuedBlocks: 4.50 KB
  MemoryUsage:
     - FreeBlocks: 704.00 KB
     - QueuedBlocks: 4.50 KB

```
Otherwise the stream load profile will not be printed to the log.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [X] Has document been added or modified
* [ ] Does it need to update dependencies
* [X] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by gitccl** [comment]
run buildall

File: be/src/common/config.h [hunk 1]
```diff
@@ -418,6 +418,8 @@ CONF_mInt32(stream_load_record_batch_size, "50");
 CONF_Int32(stream_load_record_expire_time_secs, "28800");
 // time interval to clean expired stream load records
 CONF_mInt64(clean_stream_load_record_interval_secs, "1800");
+// Whether to enable stream load profile to be printed to the log, the default is false.
+CONF_mBool(enable_stream_load_log, "false");
```
Comment (freemandealer) [inline be/src/common/config.h:422]: `enable_stream_load_profile_log`?

**Review by freemandealer — COMMENTED** [review]
Well done & thanks for your contribution. Could you provide the test results for enabling/disabling `enable_profile` & `enable_stream_load_log`  and their combinations?

File: be/src/common/config.h (hunk 1 above)
Comment (gitccl) [reply be/src/common/config.h]: done

**Comment by gitccl** [comment]
run buildall

**Review by freemandealer — APPROVED** [review]
LGTM

**Comment by gitccl** [comment]
run buildall

**Comment by gitccl** [comment]
run buildall

**Comment by freemandealer** [comment]
run p0

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**